protonvpn_cli/utils.py

```diff
@@ -302,6 +302,17 @@ def make_ovpn_template():
 
     logger.debug("remote and proto lines removed")
 
+    # Issue #162
+    if os.path.isfile("/proc/sys/net/ipv6/conf/all/disable_ipv6"):
```

Comment:
If the file does not exists, doesn't it mean that IPv6 is not supported by the system and hence also in this case we need to add the pull-filters ?

Reply:
True as well. I'll rework this tomorrow with discussed changes.